dev-meetings/2025/2025-08-01/README.md

```diff
@@ -1,8 +1,117 @@
 # Gammapy Developer Meeting 
- * Friday, August 01, 2025, at 2 pm (CET) 
- * Gammapy Developer Meeting on Zoom (direct link on Slack) 
+ * Friday, August 01, 2025, at 2 pm (CET) 14:05 - 15:17
+ * Gammapy Developer Meeting on Zoom (direct link on Slack)
+
+Attendees: Régis Terrier (RT), Hanna (H), Quentin Remy (QR), Daniel Morcuende (DM), Kirsty Feijen (KF), Atreyee Sinha (AS), Katharina Egg (KE), Axel Donath (AD), Leander Schlegel (LS)
 # Agenda
 
+- RT: Yesterday we had extra meeting. Look at list of open PRs for v.2.0.
+
+ #5954: "Separate internal EventList data model from gadf format" 
+ -------
+ - RT: I Can quickly give the status. Now there is internal model for table. Conversion to and from GADF done in IO class    (EventListReader/Writer). One issue from validation error that made test fail, now added something for this: On read the reader opens table and it is checked that minimal columns are present, not a full validation but that minimal four columns are there. If not, error is raised. Otherwise, proceed building the objects and build table, add additional optional columns.
+ - QR: Can we still read the SWGO data?
+ - RT: I think it works, because the test was running on my machine. There is a test for this specific functionality. If you try to acces events, it would raise a ValueError (in case it fails ???).
```

Comment:
```suggestion
 - RT: I think it works, because the test was running on my machine. If you try to access poorly formatted events, it would raise a ValueError, though.
```